doc/examples/sample-ml-app/sample-ml-app/sample-ml-app.vcxproj

```diff
@@ -104,15 +104,15 @@
       <SDLCheck>true</SDLCheck>
       <PreprocessorDefinitions>_DEBUG;_CONSOLE;%(PreprocessorDefinitions)</PreprocessorDefinitions>
       <ConformanceMode>false</ConformanceMode>
-      <AdditionalIncludeDirectories>C:\mlpack\armadillo-8.500.1\include;C:\mlpack\mlpack-3.4.2\build\include;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
+      <AdditionalIncludeDirectories>C:\mlpack\armadillo-8.500.1\include;C:\mlpack\mlpack-4.0.0\build\include;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
```

Comment:
@rcurtin if I recall correctly, mlpack now requires the minimum version of armadillo as 9.800 instead of 8.500

Reply:
Yes, that is true.  I'm working on testing this on a Windows system (i.e. currently trying to bring up a Virtualbox instance ...), but, making these changes will likely cause a merge conflict in this PR, and if I resolve that mlpack-bot can't do its job doing the release correctly.  So, it's likely that I'll have to open a PR with the fixes for the Windows build, then another PR with the release itself.  (That's not a problem or anything, I'm just writing it as an FYI.)